shared/profile/wot-author/index.tsx

```diff
@@ -356,49 +547,55 @@ const WotModal = (props: WotModalProps) => {
   )
 }
 
-const VerificationChoice = (props: {
-  children?: React.ReactNode
+/* eslint-disable sort-keys */
```

Comment:
why?

Reply:
They are in order by quality. The same list exists elsewhere in the service and on the client in this sensible order. This makes it easier to see which one is missing or make sure they are all present. Sort keys is despicable.